var/spack/repos/builtin/packages/nsimd/package.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Nsimd(CMakePackage):
+
+    """NSIMD is a vectorization library that abstracts SIMD programming.
+    It was designed to exploit the maximum power of processors
+    at a low development cost."""
+
+    homepage = "https://agenium-scale.github.io/nsimd/"
+    url      = "https://github.com/agenium-scale/nsimd/archive/v1.0.tar.gz"
+
+    maintainers = ['eschnett']
+
+    version('1.0', 'a692d25ed0335f1a6bb901c4b2c9fc47e26f8bf3')
+
+
+
+    variant('simd',
+            default='none',
+            description='SIMD instruction set',
+            values=(
+                'none',
+                'CPU',
+                'SSE2', 'SSE42', 'AVX', 'AVX2', 'AVX512_KNL', 'AVX512_SKYLAKE',
+                'NEON128', 'AARCH64', 'SVE',
+            ),
+            multi=False)
+    variant('optionals',
+            default=(),
+            description='Optional SIMD features',
+            values=('FMA', 'FP16'),
+            multi=True)
```

Comment:
@alalazo can these values be queried from the new microarchitecture support now?

Reply:
@alalazo Can they:

Reply:
@adamjstewart @alalazo ping?